From 8557c8a3823b341607e16048d8318a1958eab3a9 Mon Sep 17 00:00:00 2001
From: Stuart Yoder <stuart.yoder@nxp.com>
Date: Thu, 12 May 2016 17:52:28 -0500
Subject: [PATCH 207/226] fsl-dpaa2: eth: add device table to driver

this is needed to have the driver loaded as a module

Signed-off-by: Stuart Yoder <stuart.yoder@nxp.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
@@ -2794,6 +2794,7 @@ static const struct fsl_mc_device_id dpa
 	},
 	{ .vendor = 0x0 }
 };
+MODULE_DEVICE_TABLE(fslmc, dpaa2_eth_match_id_table);
 
 static struct fsl_mc_driver dpaa2_eth_driver = {
 	.driver = {
